contracts/SaleShare.sol

```diff
@@ -129,7 +129,12 @@ contract SaleShare is IERC677Receiver, Ownable, ReentrancyGuard {
     function getUserInfo(address _user) public view returns (uint, uint, uint, uint, uint) {
         UserInfo memory userInfo = userInfos[_user];
 
+        if (totalAmount == 0) {
+          return (0, 0, 0, 0, 0);
```

Comment:
We should probably still return `userInfo.amount` as first argument, the UI will want to use that before the sale ends / or is finalized